commandapi-core/src/main/java/dev/jorel/commandapi/executors/ExecutionInfo.java

```diff
@@ -0,0 +1,6 @@
+package dev.jorel.commandapi.executors;
+
+import org.bukkit.command.CommandSender;
+
+public record ExecutionInfo<Sender extends CommandSender>(Sender sender, CommandArguments args) {
```

Comment:
This class, as well as its parameters (because it's a record) need JavaDocs so developers know what these are.